### ISSUE TITLE
The egs++ angular spread source has unintuitive input

### ISSUE THREAD

**Issue by rtownson** [nrc-cnrc/EGSnrc#1214]
To define the Gaussian by FWHM, it takes a negative `sigma` input. We might as well just add a `fwhm` input to make this more user friendly.

**Comment by ftessier** [comment]
Yes, I fully agree! Eventually, we ought to get rid of all cases where negative numbers are used as flags (yes, this includes negative energies for new histories in phase space files). But let's add `fwhm` right away! Now we do have to guard against negative "sigmas" anyways, so we might add a fatal error message that includes a note about this change of behaviour!

**Comment by cptfinch** [comment]
I'd like to take this on. Planning to add a `fwhm` input key and make negative `sigma` a fatal error with a message pointing to the new key, as suggested by @ftessier.

**Comment by ftessier** [comment]
Resolved with #1384.
